### PR TITLE
feat: 403 forbidden screen development

### LINKED ISSUE #1519: ⏫ 403 forbidden screen development [FIRST ERROR SCREEN]
**Describe the task**
Developing the error screen that appears in SPAR when a user does not have the authority level to view the content they're trying to access.

**Acceptance Criteria**
- [x] Refactor the `<Layout />` component so the `<Header />`'s `<SideNav />` is not showing
- [ ] Peer review with a developer
- [x] QA review with designers
- [ ] Merge PR

**Additional context**

**Text for the smallest screen breakpoint (320 px):**

Access denied
You don't have permission to access this page.

**Text for every other breakpoint:**

Access denied

Oops! This cone is protecting this area.
You don't have permission to access this page.

---

The Figma file with screen designs for reference can be found [here](https://www.figma.com/design/XpF15ggH9Gn7FeXa9zTNYm/SPAR---404-and-403-errors?node-id=4001-19244&t=f6cpw1jAUnQRrejv-1).

This is the .svg image file:

Image

### PR DESCRIPTION
**Pull request by mgaseta**
# Description
Closes #1519 

To access the new page, you can directly type in `/403` at the end of the URL, or try to access a seedlot that it's not under your role.

#### New
- Added the new page
- Created a new context to navigate to error pages without being inside the router environment
- Added the new image
- Created a new component to create the Query Client

#### Changed
- The layout for when necessary, removing the side menu, but leaving the header

#### Removed
- Nothing :)

### How was this tested?
- [ ] 🧠 Not needed
- [x] 👀 Eyeball
- [ ] 🤖 Added tests

###  What gif/image best describes this PR or how it makes you feel?

---

Thanks for the PR!

Deployments, as required, will be available below:
- [Frontend](https://nr-spar-12-frontend.apps.silver.devops.gov.bc.ca/)
- [Backend](https://nr-spar-1612-backend.apps.silver.devops.gov.bc.ca/swagger-ui/index.html)
- [Oracle-API](https://nr-spar-1612-oracle-api.apps.silver.devops.gov.bc.ca/actuator/health)

Please create PRs in draft mode.  Mark as ready to enable:
- [Analysis Workflow](https://github.com/bcgov/nr-spar/actions/workflows/analysis.yml)

After merge, new images are deployed in:
- [Merge Workflow](https://github.com/bcgov/nr-spar/actions/workflows/merge.yml)

### CODE REVIEW

File: frontend/src/views/ErrorViews/FourOhThree/styles.scss [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  @media only screen and (max-width: 1055px) {
+    margin-top: 0;
+    margin-left: 0;
+    img {
+      height: 50vh;
+      width: 90vw;
+    }
+  }
+
+  @media only screen and (max-width: 672px) {
```
Comment (ngunner15) [inline frontend/src/views/ErrorViews/FourOhThree/styles.scss:43]: Should the breakpoint be 320 px?

Comment (mgaseta) [reply frontend/src/views/ErrorViews/FourOhThree/styles.scss:43]: That's not necessary, the small breakpoint will be between 320px and 672px, anything smaller than 320px will already be in place (besides, I don't think there are many screens smaller than 320ps XD).

Comment (ngunner15) [reply frontend/src/views/ErrorViews/FourOhThree/styles.scss:43]: Got it, just wanted to confirm 😊

**Review by ngunner15 — APPROVED** [review]

**Review by RMCampos — APPROVED** [review]
LGTM

**Pull request merged**